webpack/assets/javascripts/react_app/common/sessionStorage.js

```diff
@@ -0,0 +1,23 @@
+if (!window.sessionStorage) {
```

Comment:
we used to do this with object.assign (see https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/bundle.js#L13). is this better approach then what we currently have?